Wrappers/Python/cil/plugins/ccpi_regularisation/functions/regularisers.py

```diff
@@ -121,9 +217,17 @@ def __init__(self, alpha=1, max_iteration=100, tolerance=0, isotropic=True, nonn
         self.max_iteration = max_iteration
         self.tolerance = tolerance
         self.nonnegativity = nonnegativity
-        self.device = device # string for 'cpu' or 'gpu'
+        self.device = device 
+
+        super(FGP_TV, self).__init__(strong_convexity_constant=strong_convexity_constant)
+
 
     def proximal_numpy(self, in_arr, tau):
+
+        if self.strong_convexity_constant>0:
+            tau /= (1 + tau*self.strong_convexity_constant)
+            in_arr /= (1 + tau*self.strong_convexity_constant)
```

Comment:
This line will modify the input data. Is this what we want? Does the user know that? We might get into trouble if we modify the input parameter.

The memory efficient option is to multiply the `in_arr` back at the end of the function to put back the same values that were passed

Reply:
Not sure I follow here, according to the [equations](https://github.com/TomographicImaging/CIL/issues/1136#issue-1113575050) we need to change the value of the `input data` and `tau`.

Reply:
Fixed